### DISCUSSION TITLE
Access files written by other applications on Android?

### DISCUSSION THREAD

**Discussion by UhSomeUsername** [tauri-apps/tauri#11090] [Q&A]
Hi, I'm building a Tauri v2 Android app that can read and write files in `/storage/emulated/0/Documents/`. My app can access the files it creates, but I can't read files that are created by other apps, like Nextcloud, in the same folder. 

I've set the following permissions in `mobile.json, but I also tried setting the actual path I am trying to use`:

```
{
  "$schema": "../gen/schemas/mobile-schema.json",
  "identifier": "run-app-mobile",
  "description": "Permissions to run the app (mobile only)",
  "windows": ["main"],
  "platforms": ["android"],
  "permissions": [
    "fs:read-all",
    "fs:read-dirs",
    "fs:read-files",
    "fs:read-meta",
    "fs:write-all",
    "fs:write-files",
    {
      "identifier": "fs:scope",
      "allow": [{ "path": "**" }]
    }
  ]
}

```

I feel like I have tried everything, but perhaps I am doing something wrong that is obvious to someone with more Tauri experience? Is it even possible to access files created by other applications? I appreciate any help that I can get.

Thanks!

**Comment by ZonderPlay** [comment]
did you find a solution?

**Reply by UhSomeUsername** [reply]
Well, sort of. This is an intentional limitation of the Android filesystem for security reasons, so the only way around it is to make sure your app creates the file. At first, I tried copying the files from my disk, but creating them during a sync from Nextcloud solved it.
